app/models/utility/cell_count_spot_checking.rb

```diff
@@ -48,11 +48,14 @@ def second_replicates
 
     private
 
-    # Returns the wells in the plate that are not empty in column-major order.
+    # Returns the wells in the plate in column-major order. Empty and failed
+    # wells are filtered out. The filtering will make sure that in case a well
+    # is failed, the well following that will be used for that in the download
+    # for spot checking.
     #
     # @return [Array<Well>] the wells in the plate that are not empty
     def filtered_wells
-      @filtered_wells ||= plate.wells.reject(&:empty?).sort_by(&:coordinate)
+      @filtered_wells ||= plate.wells.reject { |well| well.empty? || well.failed? }.sort_by(&:coordinate)
```

Comment:
are 'cancelled' wells possible? just wondering...

Reply:
That is a very good point. It was never mentioned in the criteria so far. I will ask and add it in a next iteration.

Reply:
Adding this comment here just in case: _"I don't think there's a GUI function to cancel a well, only fail. You can cancel the whole plate but then you wouldn't need the file."_